lisa-utils/src/main/scala/lisa/utils/KernelHelpers.scala

```diff
@@ -179,7 +179,7 @@ trait KernelHelpers {
 
   extension (judgement: SCInvalidProof) {
     def errorMsg: String =
-      s"""Failed to prove
+      s"""Failed to prove∀'x. 'P('x) ⇒ 'P('f('x))
```

Comment:
Accidentally pasted formula here, @SimonGuilloud ?

Reply:
Absolutely no idea where that comes from, thanks for the catch!